compiler/test/src/test/java/org/apache/nemo/compiler/frontend/beam/transform/GroupByKeyAndWindowDoFnTransformTest.java

```diff
@@ -57,13 +56,19 @@ private void checkOutput(final KV<String, List<String>> expected, final KV<Strin
     final List<String> expectedValue = new ArrayList<>(expected.getValue());
     result.getValue().iterator().forEachRemaining(resultValue::add);
     Collections.sort(resultValue);
+    Collections.sort(expectedValue);
 
     assertEquals(expectedValue, resultValue);
   }
 
 
-  // [---- window1 --------]         [--------------- window2 ---------------]
-  // ts1 -- ts2 -- ts3 -- watermark -- ts4 -- watermark2 -- ts5 --ts6 --ts7 -- watermark7
```

Comment:
Can we keep the existing test (fixed window), and introduce the new test (sliding window) as a separate one?

Reply:
Fixed window is a special case of sliding window, so I just added the sliding window test to test GBKW.